RepresentsMe/ViewControllers/Events/EventDetails/EventCreateViewController.swift

```diff
@@ -22,57 +22,41 @@ class EventCreateViewController: UIViewController,
                                  LocationSelectionDelegate,
                                  DatePopoverViewControllerDelegate {
 
-    @IBOutlet weak var eventImageView: UIImageView!
-    @IBOutlet weak var eventNameTextField: UITextField!
-    @IBOutlet weak var selectOfficialButton: UIButton!
-    @IBOutlet weak var selectLocationButton: UIButton!
-    @IBOutlet weak var selectDateButton: UIButton!
-
+    // MARK: - Properties
     var event: Event?                               // The Event if editing
     var selectedDate: Date?                         // The selected date
     var selectedOfficial: Official?                 // The selected Official
     var selectedLocation: CLLocationCoordinate2D?   // The selected location
     var delegate:EventListDelegate?                 // The delegate to update
+    var mapViewAnnotation:MKAnnotation?             // The dropped pin on the mapview
+    let regionInMeters:CLLocationDistance = 7500
 
+    // MARK: - Lifecycle
     /// Sets up the view for the Event if editing an Event
     override func viewDidLoad() {
         super.viewDidLoad()
-        
-        // Setup the image view
-        self.setupImageView()
 
         // If editing an Event, setup for that Event
         if let event = self.event {
             self.setupFor(event: event)
         }
-    }
 
-    /// Update the views when an Official is selected.
-    /// Implements OfficialSelectionDelegate
-    ///
-    /// - Parameter official:   the Official that was selected
-    func didSelectOfficial(official: Official) {
-        self.set(official: official)
+        self.setupMapView()
+//        self.setupLabels()
```

Comment:
Remove commented out code. Should delete `EventCreateViewController#setupLabels()` if no longer using it.